README.md

```diff
@@ -56,6 +64,51 @@ wrapStore(store, {portName: 'MY_APP'}); // make sure portName matches
 
 That's it! The dispatches called from UI component will find their way to the background page no problem. The new state from your background page will make sure to find its way back to the UI components.
 
+### 3. Optional: Implement actions whose logic only happens in the background script (we call them aliases)
+
+```js
+// background.js
+
+import { applyMiddleware, createStore } from 'redux';
+import { alias, wrapStore } from 'react-chrome-redux';
+
+const aliases = {
+  'user-clicked-alias': () => {
+    // this call can only be made in the background script
+    chrome.notifications.create(...);
+
+  };
+};
+
+const store = createStore(rootReducer,
+  applyMiddleware(
+    alias(aliases)
+  )
+);
+```
+
+```js
+// content.js
+
+import { Component } from 'react';
+
+window.store = ...; // a proxy store
+
+class ContentApp extends Component {
+  render() {
+    return (
+      <input type="button" onClick={ this.dispatchClickedAlias.bind(this) } />
+    );
+  }
+
+  dispatchClickedAlias() {
+    window.store.dispatch({ type: 'user-clicked-alias' });
+  }
+}
+```
+
+Sometimes you'll want to make sure the logic of your action creators happen in the background script. In this case, you will want to create an alias so that the alias is proxied from the content script and the action creator logic executes in the background script.
```

Comment:
I feel like this would flow better at the top of this section (above the code) instead of the bottom. What do you think?

Also, I've been using the terminology "UI Component" describe content scripts and UI pages. Pretty generic, I know, but it is a singular term to describe all the different UI pieces a Chrome Extension has.